lib/internal/streams/stream_base.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
```

Comment:
Could you move this file into `lib/internal/`? The `streams/` directory is currently used almost exclusively for the `stream` module itself, but this implements parts of network & HTTP/2 streams

Reply:
aha i see, makes sense. doing it right now.